rust/migrate-wicked/tests/test.sh

```diff
@@ -67,6 +130,7 @@ for test_dir in ${TEST_DIRS}; do
     fi
 
     for cmp_file in $(ls -1 $test_dir/system-connections/); do
+        log_verbose "RUN: diff --unified=0 --color=always -I uuid -I timestamp $test_dir/system-connections/$cmp_file /etc/NetworkManager/system-connections/${cmp_file}"
```

Comment:
Maybe also `log_verbose` the full contents of both of these files? So you not only see the diff, but also the full content and compare it directly. I would find this useful, because sometimes the diff output can be a bit hard to interpret, WDYT?

Reply:
What about using 
```
diff -y $a  $b
```
or 
```
diff -U 1000000 $a $b
```
To see the full context?

Reply:
Sure, I personally like `diff -y`, but I think it doesn't work well with the CI, because the output may be too wide, but if you put that into `log_verbose`, sure.

Reply:
Used `-y`. But it doesn't have colors, still. When I use `-U 100000` the `-I` arguments are not working anymore...